pages/tutorials/Quick-Start-Guide.md

```diff
@@ -0,0 +1,95 @@
+> This guide is intended for the **Bukkit** version of Terra. Guides will be written for other platforms once they
+> mature.
+
+> If you have already installed Terra or know how to install Bukkit plugins, you can skip to
+> [Setting up a World](#setting-up-a-world).
+
+### Download & Installation
+
+1. Download the latest stable Terra Bukkit release from the SpigotMC website
+[here](https://www.spigotmc.org/resources/terra.85151/).
+
+2. Once you have downloaded the `.jar` file from Spigot, simply place the file your `plugins` folder located inside your
+server root.
+
+3. Once the plugin has been installed start your server up.
```

Comment:
"start your server up" replace with "start your server"

should also probably put a comma before that phrase.